ldaptor/encoder.py

```diff
@@ -11,3 +13,23 @@ def to_bytes(value):
     if isinstance(value, six.text_type):
         return value.encode('utf-8')
     return bytes(value)
+
+
+def to_unicode(value):
+    if isinstance(value, six.binary_type):
+        return value.decode('utf-8')
+    return value
+
+
+class WireStrAlias(object):
```

Comment:
missing docstring/documentation for this class.

When, why and how to use this class :)

I saw the comment in the PR description, but people are reading the code :)

In an ideal case, the PR description will be empty and the code and tests will speak for the changes :)

Reply:
Thanks! The description was added.